ui/src/seminars/components/SeminarDatesTimes.jsx

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import moment from 'moment-timezone';
+import { DISPLAY_TIME } from '../../common/constants';
+
+function SeminarDatesTimes({
+  startDate,
+  endDate,
+  timezone,
+  displayTimezone,
+  className,
+}) {
+  const startMoment = moment.utc(startDate).tz(timezone);
+  const endMoment = moment.utc(endDate).tz(timezone);
+  const DATE_AND_TIME_DISPLAY_FORMAT = `D MMMM YYYY, ${DISPLAY_TIME.format}`;
+  const startDateDisplay = startMoment.format(DATE_AND_TIME_DISPLAY_FORMAT);
+  const endDateDisplay = startMoment.isSame(endMoment, 'day')
+    ? endMoment.format(DISPLAY_TIME.format)
+    : endMoment.format(DATE_AND_TIME_DISPLAY_FORMAT);
+  return (
+    <span className={className}>{`${startDateDisplay} - ${endDateDisplay}${
```

Comment:
it's weird that whole children is a template string which is not needed in `JSX` you can just

```jsx
<span ...>
 {startDateDisplay} - {endDateDisplay} {displayTimezone && moment.tz(timezone).format('z')}
</span>
```